tests/websocket_server.py

```diff
@@ -0,0 +1,60 @@
+import threading
+
+import pytest
+from ws4py.server.wsgirefserver import WebSocketWSGIRequestHandler, WSGIServer
+from ws4py.server.wsgiutils import WebSocketWSGIApplication
+from wsgiref.simple_server import make_server
+
+
+class WebsocketServer(threading.Thread):
```

Comment:
This would be in use in the tests of the WSClient, another PR to not prolong this one